GameData/RP-0/Tree/TREE-Parts.cfg

```diff
@@ -25359,3 +25359,197 @@
+}
+
+@PART[ca_vor_tank]:FOR[xxxRP0]
+{
+	%TechRequired = materialsScienceAdvCapsules
+    %cost = 100
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_mptank]:FOR[xxxRP0]
+{
+	%TechRequired = materialsScienceAdvCapsules
+    %cost = 100
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_sep1]:FOR[xxxRP0]
+{
+	%TechRequired = materialsScienceAdvCapsules
+    %cost = 50
+    %entryCost = 100
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_sp*]:FOR[xxxRP0]
+{
+	%TechRequired = improvedPower
+    %cost = 375
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_sar]:FOR[xxxRP0]
+{
+	%TechRequired = deepSpaceScience
+    %cost = 1000
+    %entryCost = 1000
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_dish]:FOR[xxxRP0]
+{
+	%TechRequired = improvedComms
+    %cost = 500
+    %entryCost = 1000
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_comm]:FOR[xxxRP0]
+{
+	%TechRequired = improvedComms
+    %cost = 150
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_camera]:FOR[xxxRP0]
+{
+	%TechRequired = scienceAdvCapsules
+    %cost = 250
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_fom_sp]:FOR[xxxRP0]
+{
+	%TechRequired = improvedPower
+    %cost = 50
+    %entryCost = 100
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_fom_parachute]:FOR[xxxRP0]
+{
+	%TechRequired = earlyLanding
+    %cost = 500
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_fom_drogue]:FOR[xxxRP0]
+{
+	%TechRequired = earlyLanding
+    %cost = 500
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_fom_heatS]:FOR[xxxRP0]
+{
+	%TechRequired = earlyLanding
+    %cost = 500
+    %entryCost = 500
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_fom_lander]:FOR[xxxRP0]
+{
+	%TechRequired = matureAvionics
+    %cost = 1000
+    %entryCost = 5000
+    RP0conf = true
+    @description ^=:$: <b><color=green>From Coatl Aerospace mod</color></b>
+}
+
+@PART[ca_vor_core]:FOR[RP-0]
+{
+	MODULE
+	{
+		name = ModuleAvionics
+		massLimit = 12.0
+	}
+}
+
+@PART[ca_fom_lander]:FOR[RP-0]
+{
+	MODULE
+	{
+		name = ModuleAvionics
+		massLimit = 2.0
+	}
+	@MODULE[ModuleRTAntenna]
```

Comment:
That's...bad. You should never really need to change any RO parameters via RP-0 or vice versa. Is there a reason for that?

Reply:
Woops, that wasn't meant to be left in there. I put it together in a separate config and had some other changes to the same parts. Missed one when I was separating the techtree stuff.

Reply:
Oh OK, that makes sense! Ping me if you feel that some RT values need update, since i am trying to push an update for CA that includes the new parts.